packages/analytics/README.md

```
@@ -0,0 +1,32 @@
+await initSegment({ segmentKey: 'write-key', enabled: true });
+
+// Fire a tracking event (parameterized — no globals)
+fireTrackingEvent('button_clicked', { buttonId: 'submit' }, {
+  clusterID: 'cluster-1',
+  devMode: false,
+  version: '1.0.0',
+});
+
+// Compute an anonymous user ID (SHA-1 hash)
+const anonId = await computeAnonymousUserId('username');
+```
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)